src/current/_includes/releases/v22.1/v22.1.22.md

```diff
@@ -0,0 +1,34 @@
+## v22.1.22
+
+Release Date: August 14, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-22-general-changes">General changes</h3>
+
+- Improved the error message that is logged when a changefeed is triggered by dropping the parent database or a type. The previous error was of the format `value type is not BYTES: UNKNOWN`. [#107943][#107943]
+
+<h3 id="v22-1-22-sql-language-changes">SQL language changes</h3>
+
+- When no data is persisted to SQL statistics tables, such as when no flush has occurred or when flushing is disabled, the endpoint's combined view is shown, which includes in-memory data. [#104052][#104052]
+
+<h3 id="v22-1-22-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where `SHOW DEFAULT PRIVILEGES` returned no information for a database with uppercase or special characters in its name. [#103954][#103954]
+- Fixed a bug where additional empty bytes were written to files in Amazon S3 when the internal cluster setting `cloudstorage.s3.buffer_and_put_uploads.enabled` is `true`. [#104132][#104132]
```

Comment:
@stevendanna PTAL

Reply:
@dt Would you mind taking a look on Steve's behalf since you reviewed the upstream change and it's evening for him?

Reply:
This can be dropped.  This bug never shipped in 22.1 and this is a hidden feature flag.

Reply:
```suggestion
```

Reply:
Done. Thanks!